test/mocks/getPackageCkanResponse.json

```diff
@@ -0,0 +1,24 @@
+{
+  "success": true,
+  "result": {
+    "name": "gdp",
+    "resources": [
+      {
+        "name": "data.csv",
+        "url": "http://someplace.com/data.csv",
+        "size": 100
+      }
+    ],
+    "description": "GDPs list",
+    "schema": {
+      "fields": [
+        { "name": "id", "type": "integer" },
+        { "name": "title", "type": "string" }
+      ]
+    },
+    "isopen": true,
+    "num_tags": 1,
```

Comment:
Where are the tags ...?

Reply:
Shouldn't it be removed? After the conversion, just reused some tests inputs from **frictionless-ckan-mapper-js** lib.